static/app/components/events/breadcrumbs/breadcrumbItemContent.tsx

```diff
@@ -45,7 +47,24 @@ export default function BreadcrumbItemContent({
 
   const defaultMessage = defined(bc.message) ? (
     <BreadcrumbText>
-      <StructuredData value={bc.message} meta={meta?.message} {...structuredDataProps} />
+      {fullyExpanded ? (
+        <StructuredData
+          value={bc.message}
+          meta={meta?.message}
+          {...structuredDataProps}
+        />
+      ) : (
+        <StructuredData
+          value={
+            bc.message.length > MESSAGE_PREVIEW_CHAR_LIMIT
+              ? bc.message.substring(0, MESSAGE_PREVIEW_CHAR_LIMIT) + '...'
```

Comment:
i've heard to prefer the unicode character "it provides a semantically meaningful symbol rather than just three generic periods"
```suggestion
              ? bc.message.substring(0, MESSAGE_PREVIEW_CHAR_LIMIT) + '\u2026'
```